pkg/virtctl/vnc/vnc.go

```diff
@@ -280,29 +280,25 @@ func checkAndRunVNCViewer(doneChan chan struct{}, viewResChan chan error, port i
 	}
 
 	if vncBin == "" {
-		glog.Errorf("No supported VNC app found in %s", osType)
+		log.Log.Errorf("No supported VNC app found in %s", osType)
 		err = fmt.Errorf("No supported VNC app found in %s", osType)
 	} else {
-		if glog.V(4) {
```

Comment:
Keep `V(4)`

Reply:
client-go/log package doesn't provide any public attribute or method to figure out the current verbosity level

Reply:
Should be fine to introduce one method that does return the current set filtered LogLevel. Please make this addition its own commit.

Reply:
Here it goes: https://github.com/kubevirt/kubevirt/pull/11644
(I was actually in need of such a function too!)

Reply:
Thinking about this, do we even need the if statement around the call to log?

Why not do it like this?

```
log.Log.V(4).Infof("Executing commandline: '%s %v'", vncBin, args)
```

Reply:
Here, not indeed. In line 314 though, using 9e5f2fc6422ed156282fff88643fb2c441a96764 can be helpful

Reply:
I don't know but I think logging here depends on which verbosity level set before logging